tests/test_tasks.py

```diff
@@ -16,6 +16,7 @@
 @pytest.fixture
 def job():
     j = Mock()
+    j.item.handle = None
```

Comment:
This is exactly what I was going to suggest. A Mock will return a new Mock for every method or attribute accessed.